extensions/fairifier-rdf-extension/deri/grefine/rdf/vocab/VocabularyImporter.java

```diff
@@ -71,27 +84,24 @@ public void importVocabulary(String name, String uri,Repository repository, List
 
 	private Repository getModel(String url,boolean strictlyRdf) throws VocabularyImportException {
 		try {
-			logger.debug("test");
-			Any23 runner;
+		        Any23 runner;
 			if(strictlyRdf){
 				runner = new Any23("rdf-xml");
 			}else{
 				runner = new Any23();
 			}
 			runner.setHTTPUserAgent("google-refine-rdf-extension");
-			HTTPClient client = runner.getHTTPClient();
-			DocumentSource source = new HTTPDocumentSource(client, url);
+                        HTTPClient client = runner.getHTTPClient();
+			DocumentSource source = new HTTPDocumentSource(client, url);			
```

Comment:
Superfluous tabs trailing this line. Have you set your editor to indent using spaces instead of tabs?